server/src/main/java/org/apache/druid/server/coordinator/DruidCoordinator.java

```diff
@@ -743,13 +756,26 @@ private void stopBeingLeader()
     // CompactSegmentsDuty should be the last duty as it can take a long time to complete
     duties.addAll(makeCompactSegmentsDuty());
 
-    log.debug(
+    log.info(
         "Done making indexing service duties %s",
         duties.stream().map(duty -> duty.getClass().getName()).collect(Collectors.toList())
     );
     return ImmutableList.copyOf(duties);
   }
 
+  private List<CoordinatorDuty> makeMetadataStoreManagementDuties()
+  {
+    List<CoordinatorDuty> duties = ImmutableList.<CoordinatorDuty>builder()
+                                                .addAll(metadataStoreManagementDuties)
+                                                .build();
+
+    log.info(
```

Comment:
Similarly, does this log need to be info? Coordinator prints a lot of logs and it's better to reduce them if possible.

Reply:
Changed back to debug.